spec/src/main/asciidoc/WebServicesMetadata.adoc

```diff
@@ -555,385 +540,264 @@ If the @WebService.targetNamespace annotation is on a service
 implementation bean that does reference a service endpoint interface
 (through the endpointInterface annotation element), the targetNamespace
 is used for only the wsdl:service (and associated XML elements).
+|Implementation-defined, as described in Jakarta XML Web Services 2.3 [5], section 3.2.
 
-a|
-Implementation-defined, as described in JAX-WS 2.0 [5], section 3.2.
-
-a|
-serviceName
-
-a|
-The service name of the Web Service. Used as the name of the
+|serviceName
+|The service name of the Web Service. Used as the name of the
 wsdl:service when mapped to WSDL 1.1.
 
 This member-value is not allowed on endpoint interfaces.
-
-a|
-Simple name of the Java class + “Service"
-
+|Simple name of the Java class + “Service"
 |===
 
 
-[cols=",,",]
+[cols=3, options=header]
 |===
-a|
-Member-Value
+|Member-Value
+|Meaning
+|Default
```

Comment:
there is no need to have two tables for one annotation, just "merge" them

Reply:
ok, resolved by https://github.com/eclipse-ee4j/jws-api/pull/69/commits/62fdf397f28a148826367da36bcaef2c30e97c7d